hypertrace-gradle-java-convention-plugin/src/main/java/org/hypertrace/gradle/java/convention/JavaConventionPlugin.java

```diff
@@ -31,6 +39,7 @@ private void createConvention(Project target) {
   }
 
   private void configureCompatibility(Project target) {
+    // TODO: resolve about usage of `.get()`
```

Comment:
`setSource(/target)Compatibility` with a property was not working

Reply:
Ah they're still eager-only? This actually reminds me though - I Thought they weren't compatible with toolchains?

Reply:
If we have in the source target compatibility set to 11, even if we set 21 toolchain it was compiling to 11. Hence I thought we need both to be configured